src/collections/infra/repositories/CollectionsRepository.ts

```diff
@@ -40,11 +53,25 @@ export class CollectionsRepository extends ApiRepository implements ICollections
       })
     )
 
+    const inputLevelsRequestBody: NewCollectionInputLevelRequestPayload[] = []
+    collectionDTO.inputLevels.forEach((element) => {
+      inputLevelsRequestBody.push({
+        datasetFieldTypeName: element.datasetFieldName,
+        include: element.include,
+        required: element.required
+      })
+    })
+
     const requestBody: NewCollectionRequestPayload = {
       alias: collectionDTO.alias,
       name: collectionDTO.name,
       dataverseContacts: dataverseContacts,
-      dataverseType: collectionDTO.type.toString()
+      dataverseType: collectionDTO.type.toString(),
```

Comment:
What is the need of doing `.toString()` on the `collectionDTO.type` property?

Reply:
No need, changed!